corehq/form_processor/models.py

```diff
@@ -174,7 +177,7 @@ def is_submission_error_log(self):
         return self.state == self.SUBMISSION_ERROR_LOG
 
     @property
-    @memoized
+    @cache_return_value
```

Comment:
I semi agree but I also feel that this is caching the work done to convert the XML to JSON